etrago/appl.py

```diff
@@ -383,6 +384,11 @@ def run_etrago(args, json_path):
             Limit overall energy production country-wise for each generator
             by carrier. Set upper/lower limit in p.u.
 
+    delete_dispensable_ac_buses: bool
+        Choose if unnecessary buses should be identified and deleted from the
+        grid. This buses have no load or generation attached. Additionally,
+        they are just connected to two other buses.
+        Default: False.
```

Comment:
I would probably not call it "unnecessary buses" that somehow sounds as if it would be a bug. 
It would be also good to mention that it affects the electrical grid.
I would suggest: 

Choose if electrical buses that are only connecting two lines should be removed.  These buses have no other components attached to them, the connected lines are merged. This reduces the spatial complexity without losing any accuracy.

Reply:
Changed.